app/assets/javascripts/actions/story.js

```diff
@@ -28,8 +39,13 @@ export const updateStory = (story, projectId) => {
   };
 };
 
-export const editStory = (id, newAttributes) => ({
-  type: actionTypes.EDIT_STORY,
-  id,
-  newAttributes
-});
+export const deleteStory = (storyId, projectId) => {
+  return (dispatch, getState, { Story }) => {
```

Comment:
I think here and bellow you could just `=> () => {}` instead of `{ return () => {} }` ... but idk if people would be concerned about readability...